pages/common/kdeconnect-cli.md

```diff
@@ -0,0 +1,35 @@
+# kdeconnect-cli
+
+> KDE Connect CLI.
+
+- List all devices:
+
+`kdeconnect-cli --list-devices`
+
+- List all reachable devices:
+
+`kdeconnect-cli --list-available`
+
+- Request pairing with a specific device, specifying its ID:
+
+`kdeconnect-cli --pair --device {{device_id}}`
+
+- Request pairing with a specific device, specifying its name:
+
+`kdeconnect-cli --pair --name {{device_name}}`
+
+- Ring a specific device:
```

Comment:
Does this cause a device to make a noise, or initiate a call with it?

Reply:
It makes a noise

Reply:
Ah, ok